cypress/integration/auth.js

```diff
@@ -41,7 +41,7 @@ describe('Auth', () => {
 
                 cy.get('#inputEmail').type(Cypress._.random(0, 1e6) + '@posthog.com')
 
-                cy.get('#inputPassword').type('password').should('have.value', 'password')
+                cy.get('#inputPassword').type('Test1234').should('have.value', 'Test1234')
```

Comment:
The frontend requirement for the password is tested here. The backend could also use an extra check and maybe be added here #1145